src/omni_epd/displays/inky_display.py

```diff
@@ -38,7 +36,11 @@ def __init__(self, deviceName, config):
         super().__init__(deviceName, config)
 
         # need to figure out what type of device we have
-        dType, dColor = deviceName.split('_')
+        deviceDetail = deviceName.split('_')

```

Comment:
If the device is `auto` would it be better to detect for this specifically and then branch into the `deviceName.split()` code? Not comprehensive but what about something like: 
```
if(deviceName == 'auto'):
  dType = deviceName
  dColor = ''
else:
  dType, dColor = deviceName.split('_')

```

Reply:
I've extracted this into a method that uses a similar approach on another branch, I'll Cherry pick that into the PR here later this eve.